releases/release-6.5.5.md

```diff
@@ -0,0 +1,85 @@
+---
+title: TiDB 6.5.5 Release Notes
+summary: 了解 TiDB 6.5.5 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 6.5.5 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：6.5.5
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v6.5/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v6.5/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v6.5.5#version-list)
+
+## 改进提升
+
++ TiDB **tw@qiancai**
+
+    - 支持从高版本 TiDB 回退到 v6.5 [#45570](https://github.com/pingcap/tidb/issues/45570) @[wjhuang2016](https://github.com/wjhuang2016)
+    - (dup): release-7.3.0.md > # 稳定性 * 新增部分优化器提示，包括 [`NO_MERGE_JOIN()`]、[`NO_INDEX_JOIN()`](/optimizer-hints.md#no_index_joint1_name--tl_name-)、[`NO_INDEX_MERGE_JOIN()`](/optimizer-hints.md#no_index_merge_joint1_name--tl_name-)、[`NO_HASH_JOIN()`](/optimizer-hints.md#no_hash_joint1_name--tl_name-)、[`NO_INDEX_HASH_JOIN()`](/optimizer-hints.md#no_index_hash_joint1_name--tl_name-) [#45520](https://github.com/pingcap/tidb/issues/45520) @[qw4990](https://github.com/qw4990)
+    - 添加 coprocessor 相关的 request source 信息 [#46514](https://github.com/pingcap/tidb/issues/46514) @[you06](https://github.com/you06)
+
++ TiKV **tw@qiancai**
+
+    - 新增 PD Client 连接重试过程中的 backoff 机制。异常错误重试期间，逐步增加重试时间间隔，减小 PD 压力 [#15428](https://github.com/tikv/tikv/issues/15428) @[nolouch](https://github.com/nolouch)
+    -避免写 Titan manifest 文件时持有锁导致影响其他线程 [#15351](https://github.com/tikv/tikv/issues/15351) @[Connor1996](https://github.com/Connor1996)
```

Comment:
```suggestion
```

Reply:
moved this note to https://github.com/pingcap/docs-cn/pull/15040/commits/677b58e389c54a994e888765ce974ba936736155